data_steward/bq_utils.py

```diff
@@ -113,3 +118,74 @@ def get_job_details(job_id):
     bq_service = create_service()
     app_id = app_identity.get_application_id()
     return bq_service.jobs().get(projectId=app_id, jobId=job_id).execute()
+
+
+def merge_tables(source_table_name_list, source_dataset_name, destination_table_name, destination_dataset_name):
+    """Takes a list of table names and runs a copy job
+
+    :source_table_name_list: list of tables to merge
+    :source_dataset_name: dataset where all the source tables reside
+    :destination_table_name: data goes into this table
+    :destination_dataset_name: dataset where the destination table resides
+    :returns: True if successfull. Or False if error or taking too long.
+
+    """
+    app_id = app_identity.get_application_id()
+    job_body = { 
+        'configuration': {
+            "copy": { # "sourceTable": { "projectId": string, "datasetId": string, "tableId": string },
+                "sourceTables": [{
+                                        "projectId": app_id,
+                                        "datasetId": source_dataset_name,
+                                        "tableId": table_name
+                        } for table_name in source_table_name_list], 
```

Comment:
I will make separate variable for readability

Reply:
Made this change.